### PR TITLE
feat: refine the docker setup script

### PR DESCRIPTION
**Pull request by tokers**
Signed-off-by: Chao Zhang <tokers@apache.org>

### CODE REVIEW

File: scripts/run-apisix-on-docker.sh [hunk 1]
```diff
@@ -177,17 +152,24 @@ download_docker_image() {
   debug "downloaded APISIX docker image"
 }
 
+download_cloud_module() {
+    debug "downloading cloud lua module ..."
+
+    /bin/bash <(curl -fsSL 'https://raw.githubusercontent.com/api7/cloud-scripts/a9fa31ae0518e5188f66b42a5e46042b75cad993/scripts/cloud-module-fetcher.sh')
```
Comment (starsz) [inline scripts/run-apisix-on-docker.sh:158]: There are two `download_cloud_module` function ?

Comment (nic-6443) [inline scripts/run-apisix-on-docker.sh:158]: Don't you need to use the master branch here?

Comment (tokers) [reply scripts/run-apisix-on-docker.sh:158]: Where is the other one?

Comment (tokers) [reply scripts/run-apisix-on-docker.sh]: Changed.

**Review by nic-6443 — APPROVED** [review]

File: scripts/run-apisix-on-docker.sh [hunk 2]
```diff
@@ -132,35 +114,28 @@ configure_certificate() {
 # generate the APISIX gateway configuration file
 generate_configuration() {
   debug "generating APISIX configuration"
+  CONFIG_PATH=`mktemp /tmp/apisix-config.yaml.XXXXXX`
```
Comment (starsz) [inline scripts/run-apisix-on-docker.sh:117]: Why add the `XXXXX` suffix?

Comment (tokers) [reply scripts/run-apisix-on-docker.sh:117]: Please run `man mktemp`.

**Review by starsz — APPROVED** [review]

**Pull request merged**